prims/prims.c

```diff
@@ -0,0 +1,47 @@
+#include <stdio.h>
+#include <string.h>
+
+#define SIZE 5
+#define INFINITY 32767
```

Comment:
What I suggested was that instead of `INFINITY`, you could use `INT_MAX` in the code.